app/storyedit/directives/edit.component.ts

```diff
@@ -1,62 +1,59 @@
 import {Component, Input} from 'angular2/core';
-import {NgForm} from 'angular2/common';
 import {StoryModel} from '../models/story.model';
+import {CATEGORIES, SUBCATEGORIES} from '../models/story.categories';
+import {StoryFieldComponent} from './storyfield.component';
 
 @Component({
-  directives: [NgForm],
+  directives: [StoryFieldComponent],
   selector: 'newstory-edit',
-  styleUrls: ['app/storyedit/storyedit.forms.css'],
+  styleUrls: ['app/storyedit/directives/edit.component.css'],
   template: `
     <form *ngIf="story && story.isLoaded">
 
-      <div [class]="fieldClasses('title')">
-        <h3><label for="title" required>Story Title</label></h3>
-        <p class="hint">Write a short, Tweetable title like a newspaper headline.
-          Make viewers want to click on your piece from the title alone.</p>
-        <textarea id="title" [(ngModel)]="story.title"></textarea>
-        <p class="error">Title {{story.invalid.title}}</p>
-      </div>
-
-      <div [class]="fieldClasses('shortDescription')">
-        <h3><label for="teaser" required>Teaser</label></h3>
-        <p class="hint">A first impression; think of this as the single-item
-          lead of a piece.</p>
-        <textarea id="teaser" [(ngModel)]="story.shortDescription"></textarea>
-        <p class="error">Teaser {{story.invalid.shortDescription}}</p>
-      </div>
+      <story-field [story]="story" textarea="true" name="title" label="Story Title" required>
+        <hint>Write a short, Tweetable title like a newspaper headline. Make viewers
+          want to click on your piece from the title alone.</hint>
+      </story-field>
+
+      <story-field [story]="story" textarea="true" name="shortDescription" label="Teaser" required>
+        <hint>A first impression; think of this as the single-item lead of a piece.</hint>
+      </story-field>
 
       <hr/>
 
-      <div class="field">
-        <h3><label required>Story Versions</label></h3>
+      <story-field label="Story Versions" required>
         <div class="uploads-box">uploads goes heres</div>
-      </div>
+      </story-field>
 
       <hr/>
 
-      <div class="field">
-        <h3><label required>Tag your Story</label></h3>
-        <p class="hint">Adding tags that are relevant to your piece helps people
-          find your work on PRX and can help you be licensed and heard.</p>
-      </div>
+      <story-field [story]="story" label="Tag your Story" invalid="tags" invalidlabel="" required>
```

Comment:
makes sense, the only other way I guess would be a distinct directive/component just for this, and that seems like overkill